loaders/yaml.js

```diff
@@ -10,6 +10,12 @@ export function parseFrontmatter(content) {
 
     if (end !== -1) {
       const data = parse(content.slice(3, end));
+
+      // Allow empty front matter
+      if (typeof data === 'undefined') {
+        return { content };
+      }
+
```

Comment:
This returns the content including the front matter delimiters. I think a better solution is:
```js
const data = parse(content.slice(3, end)) || {};
```

Reply:
Do we want the following to cause an error?

```
---
0
---
```

That can be done by using:

```js
const data = parse(content.slice(3, end)) ?? {};
```

But it still allows the following:

```
---
null
---
```

Reply:
Yes, an invalid yaml should cause an error.

Reply:
I changed it to the following, which allows an empty front matter (which gets parsed to `undefined`) but not any other falsy values (such as `null`).

```js
if (typeof data === 'undefined') {
  data = {};
}
```

Reply:
Nice!